### PR TITLE
Integrate srml/im-online

### PR DESCRIPTION
**Pull request by cmichi**
This is the basic version of the module which was merged a week ago.

There will be some follow-up PR when the module supports integration with slashing.

### CODE REVIEW

File: service/src/chain_spec.rs [hunk 1]
```diff
@@ -126,6 +127,10 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 		curated_grandpa: Some(CuratedGrandpaConfig {
 			shuffle_period: 1024,
 		}),
+		im_online: Some(ImOnlineConfig{
+			gossip_at: 0,
+			last_new_era_start: 0,
```
Comment (cmichi) [inline service/src/chain_spec.rs]: We can remove those two properties from the config in the next version of the module.

**Comment by ltfschoen** [comment]
i had a quick look and it looks ok from a layman's perspective

File: service/src/chain_spec.rs [hunk 2]
```diff
@@ -256,6 +261,10 @@ pub fn testnet_genesis(
 		curated_grandpa: Some(CuratedGrandpaConfig {
 			shuffle_period: 1024,
 		}),
+		im_online: Some(ImOnlineConfig{
```
Comment (rphmeier) [inline service/src/chain_spec.rs]:
```suggestion
		im_online: Some(ImOnlineConfig {
```

File: service/src/chain_spec.rs [hunk 3]
```diff
@@ -126,6 +127,10 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
...
```
Comment (rphmeier) [inline service/src/chain_spec.rs]:
```suggestion
		im_online: Some(ImOnlineConfig {
```

what's with the missing spaces habit?

Comment (cmichi) [reply service/src/chain_spec.rs]: Thanks for catching it. Sometimes, in my wildest dreams, I dream of a linter in the CI.

**Comment by kianenigma** [comment]
closed via #353

**Pull request closed without merging**